### PR TITLE
feat(LUX-29): new component: login-method-button

### PR DESCRIPTION
**Pull request by remypar5**
For reviewers:
- The name of the component became the topic of discussion again. Since this is a button which directs you to a "middel" instead of a "manier", the term `method` seems unsuitable. If you have a suggestion, please provide it.
- There are no tests, yet, and are out of scope for this PR. 
- Currently, Storybook's [preview.tsx](packages/storybook/config/preview.tsx) shows both dark and default/light mode. That will change later to a theme switcher. 
- The core tokens have the `logius` namespace whereas the common and component tokens have the `lux` namespace. This is deliberate
- The component catches and stops the HTML `click` event and replaces it with a `luxClick` event. This is to make sure the internals of the component (which is protected by the shadowDom) don't get exposed. We are still debating whether or not to do this. 
-

### CODE REVIEW

File: packages/components-css/login-method-button/README.md
Comment (dleeuwen) [inline packages/components-css/login-method-button/README.md:1]: Ik zou ook zoiets als auth-button gebruiken, agent zegt me niet direct iets.

**Comment by rwittenberns** [comment]
De CI pipeline faalt nog op het `pnpm run build:style-dictionary` command

**Review by rwittenberns — CHANGES_REQUESTED** [review]

**Comment by remypar5** [comment]
Een handmatige review is uitgevoerd door een UX designer uit een ander team. De volgende bevindingen moeten worden behandeld:

- Icon
    - [ ] alignment/padding
    - [ ] Icon is te groot
    - Digid icon
        - [ ] border color lijkt af te wijken van design
        - [ ] border radius is korrelig
- Focus state
    - [x] outline color (light)
    - [x] outline color (dark)
    - [x] wordt ook toegepast bij active state
- Border color
    - [x] alle states (light)
    - [x] alle states (dark)
- [ ] line-height is niet gezet
- [x] Labels
    - [v] eIDAS
    - [v] Eherkenning

File: packages/components-react/src/index.test.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import { describe, expect, it } from '@jest/globals';
+
+describe('index', () => {
+  it('has a placeholder test', () => {
```
Comment (rwittenberns) [inline packages/components-react/src/index.test.ts:4]: We kunnen werken met test.todo placeholders. Op die manier laat de console output van de unit test fase zien waar de placeholders/todo's staan en verdwijnen ze niet uit het zicht.
https://jestjs.io/blog/2019/01/25/jest-24-refreshing-polished-typescript-friendly#testtodo

Comment (remypar5) [reply packages/components-react/src/index.test.ts]: Mooie vondst! Ik heb het aangepast

**Review by rwittenberns — DISMISSED** [review]

**Review by rwittenberns — APPROVED** [review]

**Pull request merged**